src/Bicep.Core.Samples/Files/TestFramework_CRLF/main.diagnostics.bicep

```diff
@@ -72,13 +79,17 @@ test testInexitentParam 'samples/main.bicep' ={
     env: 'dev'
     suffix: 10
     location: 1
+//@[04:12) [BCP037 (Error)] The property "location" is not allowed on objects of type "params". No other properties are allowed. (CodeDescription: none) |location|
   }
 }
 
 test testEmptyBody 'samples/main.bicep' = {}
+//@[05:18) [BCP035 (Error)] The specified "test" declaration is missing the following required properties: "params". (CodeDescription: none) |testEmptyBody|
 
 test testShouldIgnoreAdditionalProperties 'samples/main.bicep' = {
+//@[05:41) [BCP035 (Error)] The specified "test" declaration is missing the following required properties: "params". (CodeDescription: none) |testShouldIgnoreAdditionalProperties|
   additionalProp: {}
+//@[02:16) [BCP037 (Error)] The property "additionalProp" is not allowed on objects of type "module". Permissible properties include "params". (CodeDescription: none) |additionalProp|

```

Comment:
Was this new diagnostic intentional? The name of the test (`testShouldIgnoreAdditionalProperties`) indicates that this shouldn't raise an error.

Reply:
I dont understand why the object type is module

Reply:
fixed it

Reply:
> I dont understand why the object type is module

The type name comes from [here](https://github.com/Azure/bicep/blob/99ab8c8567e501c792b06e6277d9346bff08ddf9/src/Bicep.Core/TypeSystem/DeclaredTypeManager.cs#L1803-L1805)

Reply:
Yes, I had a typo, 

It is working now.